server/api/user.js

```diff
@@ -0,0 +1,125 @@
+var express = require('express');
+var router = express.Router();
+const { User, Cart, Order } = require('../../db');
+const { generateSessionId } = require('../utils');
+
+
+//return a single user by id
+//includes cart and orders
+router.get('/:userId', (req, res, next) => {
+    User.findOne({
+        where: {
+            id: req.params.userId,
+        },
+        include: [
+            {
+                model: Cart,
+            },
+            {
+                model: Order,
+            }
+        ]
+    })
+    .then(result => {
+        if (result) {
+            return res.status(200).send(result);
+        }
+        res.status(404).send('Not found');
```

Comment:
Should it be "User not found"? We would like to have descriptive logging for API routes.

Reply:
Logging and debugging is driven through the console, where we get a central source for messages. Descriptive responses from the server doesn't serve us much for debugging. 
I'll add console logging for the error cases.